modules/ROOT/pages/create-profiles.adoc

```diff
@@ -72,61 +72,90 @@ If you filter by categories, you can see the number of categories for each API,
 If you select the *API Instance* filter *Include only APIs with instances*, only specifications that have an instance in API Manager are included. You can then optionally filter by environment type and environment name.
 |
 Set Notifications
-|Enable or disable automatic notifications for this profile. The default is enabled. See xref:configure-notifications.adoc#send-auto-notifs[Sending Conformance Notifications].
+|Enable or disable automatic notifications for this profile. The default setting is enabled for active profiles. Notifications are automatically disabled for draft profiles. See xref:configure-notifications.adoc#send-auto-notifs[Sending Conformance Notifications].
 |Specify General Information 
 |Enter a profile name and description that helps others understand the kinds of governance rulesets included and the kinds of APIs validated in this profile.
 |Review
-|Review your profile and click *Create*.
+|Review your profile and click *Save as a draft* or *Save as active*:
+
+* Save a profile as draft to test profile settings and review conformance information before revealing the conformance information for its targeted APIs in integrating Anypoint Platform products.
```

Comment:
"in integrating" is correct, yeah?

Reply:
I'll reword to be more clear.

Reply:
I've reworded as follows:
Save a profile as draft to test profile settings and review conformance information before showing the conformance information for its targeted APIs outside of the draft profile view.

It's a bit more accurate than the other wording and hopefully more understandable. Please see what you think.